### ISSUE TITLE
Fixes of bugs / flow

### ISSUE THREAD

**Issue by KostyaEver** [ever-co/ever-teams#559]
- [x]  Invite user - cause Error for Team page (on safari browser) only when everyone joined team it displays page (probably problem to display pending users cards)
- [x]  Invite user - in the email box, the user clicks on the link it should redirect to Gauzy Teams - Team page (NOT Gauzy Platform)
- [x] Invited user - Team page invited 'User B' can't see on 'User A' card which task is assigned (looks like nothing is assigned (Only after page update. when each member tracked a bit time otherwise empty user cards only)),  also does not see 'Total time' worked today 
- [x] Invited user - Team page invited 'User B' can't see the some issue type as 'User A' in the main input. Example for User B - task 1 has 'Bug' issue type, for User A - task 1 has 'Task' issue type - in the same team
- [x]  Invited user - User B cannot see ( ... ) menu of User A
- [x]  user log out and log in - User's card is empty - not task ever is was in progress before log out, should remember the last user card condition
- [x] Team page - Verification - right after user provided the verification Code, it should display the normal team page without 'Verify' notification
- [x]  Button ‘+ Create’ should be the same length as two Tabs ‘Open’ & ‘Closed’
- [x]  Team page / task card - change ‘Priority’ & ‘Size’ labels to icons like in the design
- [x]  Team page / task card - add ‘Priority’ & ‘Size’ new icons to the end of the Task Title
- [x]  Team page / task card - remove user’s email under the Name
- [x]  Task page / task card - under Task title remove ‘Status’
- [x]  Task page / task card - change ‘Priority’ & ‘Size’ labels to icons like in the design
- [x]  Task page / task card - add ‘Priority’ & ‘Size’ new icons to the end of the Task Title
- [x]  Team page / task card - issue 'icon' should be with normal size
- [x]  Task and Team page / task card  and main input - height of labels are bigger that Task Title, should be the same or smaller
- [x]  Team page / main input - issue type should be saved once selected and task created
- [x]  Team page / main input - once user creates next task (typing the name of new in the main input, user can still see the issue type and number of previous) the problem is that user has to create task only the same issue type as the previous and then change another type that is not good.
Solution: user should be able to delete from the main input even number of previous task so that user can start typing new title and select issue type - the system understands user creates new task and gives new number
- [x]  Team page / main input - issue type + number before Title should be displayed in combobox
- [x]  Team page / main input - issue type should be changed in the task card when user changes it. Note: that ‘Epic’ if the user created can’t be changed later.
- [x] Team page - Restrict the number of symbols for a Team Name (no I can see 24 - should be no more than 10 + ... )
- [x]  Team page / main input and task card ‘Estimate part - NO need to display ‘_ _ ‘ under digits if there are digits if estimate h & m inputs are empty should be displayed  ‘_ _ ‘
- [x]  Team page - / task card - ‘Estimate’ ‘Edit’ icon is big - should be smaller if possible
- [x]  Team / Task page - ( … ) menu - let’s add ‘Task Details’ of the top of all
- [x]  Team page / Combobox - select ‘Status’ is in uncollapsed condition nested within Combobox - would be nice to see it over combobox
- [x]  Team page / Verify message - user still can create new team and invite user - Must NOT and user can’t close notification until the verified identity
- [x]  Task page / Created own task - should be displayed only in Assigned Tab and if user tracked even one sec in Worked Tab. in Tab Unassigned should be only Unassigned task to current user - not we have there any tasks
- [x]  Task page - Filter - should filter tasks according to settings
- [x]  Team page - The header of user card columns should be aligned (be precisly above columns)
- [x]  Team / Task pages don’t see Statuses / Labels etc created in Settings - should be an equal set of it
- [x]  Task page / Unassigned tab - should display number of Assignees + avatars instead of estimate column
- [x]  Task page / Unassigned tab - should display button ‘Play’ in ‘time’ column
- [x]  Task page / Unassigned tab - should display in column ‘Time’ only ‘Today’ time
- [x]  Team page / (…) menu / Edit - the cursor focus should be in at the end of the task title input on card right after user clicked ‘Edit’
- [x]  Team page / Add Tooltips ‘Please Verify your account before start using the app’ when user trying to
          - Create a Task
          - Invite a user
          - Create a Team
